com.unity.multiplayer.transport.utp/package.json

```diff
@@ -1,18 +1,12 @@
 {
   "name": "com.unity.multiplayer.transport.utp",
-  "displayName": "Unity Transport for MLAPI",
+  "displayName": "Unity Transport for Netcode for GameObjects",
+  "description": "This package is plugging Unity Transport into Netcode for GameObjects, which is a network transport layer - the low-level interface for sending UDP data",
   "version": "0.0.1-preview.1",
   "unity": "2020.3",
-  "unityRelease": "0f1",
```

Comment:
Was is intentional to drop a bunch of these fields?

Reply:
they were intentional (optional fields). the potentially important one was `"unityRelease": "0f1"` minor version field is optional and generally guided for specific minor version requirements which don't have any (we were already 0f1+).